src/main/java/org/folio/consortia/service/impl/UserAffiliationServiceImpl.java

```diff
@@ -52,7 +59,6 @@ public void createPrimaryUserAffiliation(String eventPayload) {
         log.warn("Primary affiliation already exists for tenant/user: {}/{}", userEvent.getTenantId(), userEvent.getUserDto().getUsername());
         return;
       } else {
-        String centralTenantId = folioExecutionContext.getTenantId();
         userTenantService.createPrimaryUserTenantAffiliation(consortiaTenant.getConsortiumId(), consortiaTenant, userEvent.getUserDto().getId(), userEvent.getUserDto().getUsername());
         if (ObjectUtils.notEqual(centralTenantId, consortiaTenant.getId())) {
           userTenantService.save(consortiaTenant.getConsortiumId(), createUserTenant(centralTenantId, userEvent), false);
```

Comment:
Changes in this PR needed, because userTenantService.save switches context to member tenant and so we need switch back to central tenant to produce CONSORTIUM_PRIMARY_AFFILIATION_CREATED.
We need to think how we can simplify context switching in future